.gitignore

```diff
@@ -36,3 +37,9 @@
 /.project
 /.pydevproject
 /.settings/
+
+# For Performance Team
+*.csv
+*.log
```

Comment:
Can we please add full paths to files when possible? For example, notice that we have these line earlier in this file:

    /docs/robottelo.log
    /robottelo.log

Adding `*.log` and other similar bits in to the `.gitignore` file makes those other lines obsolete and is unnecessarily broad. A user could write up some bad code that starts spewing out `*.log` or `*.csv` files or something else, and they'd never know because of this line. Or something else even more exotic.